modules/apps/digital-signature/digital-signature-test/src/testFunctional/tests/DigitalSignatureSiteSettings.testcase

```diff
@@ -37,159 +39,65 @@ definition {
+		task ("Given the digital signature using the Always Inherit strategy") {
+			DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
+		}
 
-		AssertElementPresent(
-			digitalSignatureIntegrationKey = ${digitalSignatureIntegrationKey},
-			locator1 = "DigitalSignatureConfiguration#DIGITAL_SIGNATURE_INTEGRATION_KEY");
+		task ("When you have it in Digital Signature") {
+			DigitalSignature.navigateToComponent(nameComponent = "Digital Signature");
+		}
 
-		AssertElementPresent(
-			digitalSignatureRsaPrivateKey = ${digitalSignatureRsaPrivateKey},
-			locator1 = "DigitalSignatureConfiguration#DIGITAL_SIGNATURE_RSA_PRIVATE_KEY");
+		task ("Then Digital Signature data persists in site settings after enabled with Always Inherit strategy") {
+			DigitalSignature.canDataPersistAfterEnabled(siteStrategy = "Always Inherit");
+		}
 	}
 
 	@description = "This is a test for LPS-131427. Verify if Digital Signature data persists in site settings after enabled with Inherit or Override strategy"
 	@priority = 5
 	test DataPersistsAfterEnabledInheritOrOverrideStrategy {
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Inherit or Override");
-
-		Navigator.openURL();
-
-		ProductMenu.gotoPortlet(
-			category = "Configuration",
-			portlet = "Site Settings");
-
-		Click(
-			key_itemName = "Digital Signature",
-			locator1 = "ListGroupItem#ITEM_TEXT");
-
-		var siteSettingsStrategy = "Inherit or Override";
-		var digitalSignatureApiUsername = ${digitalSignatureApiUsername};
-		var digitalSignatureApiAccountId = ${digitalSignatureApiAccountId};
-		var digitalSignatureAccountBaseUri = ${digitalSignatureAccountBaseUri};
-		var digitalSignatureIntegrationKey = ${digitalSignatureIntegrationKey};
-		var digitalSignatureRsaPrivateKey = ${digitalSignatureRsaPrivateKey};
-
-		AssertElementPresent(
-			locator1 = "ClickToChatConfiguration#SITE_SETTINGS_STRATEGY_CHOOSEN",
-			siteSettingsStrategy = ${siteSettingsStrategy});
-
-		AssertElementPresent(locator1 = "ClickToChatConfiguration#CHECK_INPUT_ENABLED");
-
-		AssertElementPresent(
-			digitalSignatureApiUsername = ${digitalSignatureApiUsername},
-			locator1 = "DigitalSignatureConfiguration#DIGITAL_SIGNATURE_USER_ID");
-
-		AssertElementPresent(
-			digitalSignatureApiAccountId = ${digitalSignatureApiAccountId},
-			locator1 = "DigitalSignatureConfiguration#DIGITAL_SIGNATURE_ACCOUNT_ID");
-
-		AssertElementPresent(
-			digitalSignatureAccountBaseUri = ${digitalSignatureAccountBaseUri},
-			locator1 = "DigitalSignatureConfiguration#DIGITAL_SIGNATURE_BASE_URI");
+		task ("Given the digital signature using the Inherit or Override strategy") {
+			DigitalSignature.enableDigitalSignature(siteStrategy = "Inherit or Override");
+		}
 
-		AssertElementPresent(
-			digitalSignatureIntegrationKey = ${digitalSignatureIntegrationKey},
-			locator1 = "DigitalSignatureConfiguration#DIGITAL_SIGNATURE_INTEGRATION_KEY");
+		task ("When you have it in Digital Signature") {
```

Comment:
Please, change this task to:`When the user goes to Digital Signature by portlet`